File: ServerSpace/src/main/java/Models/Servicio/ServicioSaveModel.java
package Models.Servicio;

import java.sql.Timestamp;
import com.fasterxml.jackson.annotation.JsonProperty;

import EntityLayer.ServicioEntity;
import Enumerados.ProcessActionEnum;

public class ServicioSaveModel {
 
   
   
    public ServicioSaveModel() {
        this.servicioId = 0;
        this.correlativo = "";
        this.nombre = "";
        this.descripcion = "";
        this.tipoServicioId= 0;
        this.fechaRegistro = null;
        this.codUsuario = "";
        this.estadoRegistro = false;
    }

    public ServicioSaveModel(ServicioEntity Ent) {
        this.servicioId = Ent.getServicioId();
        this.correlativo = Ent.getCorrelativo();
        this.nombre = Ent.getNombre();
        this.descripcion = Ent.getDescripcion();
        this.tipoServicioId= Ent.getTipoServicioId();
        this.fechaRegistro = Ent.getFechaRegistro();
        this.codUsuario = Ent.getCodUsuario();
        this.estadoRegistro = Ent.getEstadoRegistro();
    }

   
    @JsonProperty("ServicioId")
    private int servicioId = 0;

    public int getServicioId() {
        return servicioId;
    }

    public void setServicioId(int servicioId) {
        this.servicioId = servicioId;
    }

    @JsonProperty("Correlativo")
    private String correlativo = "";

    public String getCorrelativo() {
        return correlativo;
    }

    public void setCorrelativo(String correlativo) {
        this.correlativo = correlativo;
    }

    @JsonProperty("Nombre")
    private String nombre = "";

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    @JsonProperty("Descripcion")
    private String descripcion = "";

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    @JsonProperty("TipoServicioId")
    private int tipoServicioId = 0;

    public int getTipoServicioId() {
        return tipoServicioId;
    }

    public void setTipoServicioId(int tipoServicioId) {
        this.tipoServicioId = tipoServicioId;
    }

    @JsonProperty("FechaRegistro")
    private Timestamp fechaRegistro = null;

    public Timestamp getFechaRegistro() {
        return fechaRegistro;
    }

    public void setFechaRegistro(Timestamp fechaRegistro) {
        this.fechaRegistro = fechaRegistro;
    }

    @JsonProperty("CodUsuario")
    private String codUsuario = "";

    public String getCodUsuario() {
        return codUsuario;
    }

    public void setCodUsuario(String codUsuario) {
        this.codUsuario = codUsuario;
    }

    @JsonProperty("EstadoRegistro")
    private boolean estadoRegistro = false;

    public boolean getEstadoRegistro() {
        return estadoRegistro;
    }

    public void setEstadoRegistro(boolean estadoRegistro) {
        this.estadoRegistro = estadoRegistro;
    }

    @JsonProperty("Action")
    private ProcessActionEnum action = ProcessActionEnum.Loaded;

    public int getAction() {
        return action.getValor();
    }

    public void setAction(ProcessActionEnum Action) {
        this.action = Action;
    }
}

File: SharedApi/Space/Space.Server/Model/Servicio/ServicioSaveModel.cs
using Space.EntityLayer;
using System.Text.Json.Serialization;

namespace Space.Server
{
    public class ServicioSaveModel
    {
        public ServicioSaveModel()
        {
            ServicioId = 0;
            Correlativo = String.Empty;
            Nombre = String.Empty;
            Descripcion = String.Empty;
            TipoServicioId = 0;
            FechaRegistro = DateTime.Now;
            CodUsuario = String.Empty;
            EstadoRegistro = true;
        }

        public ServicioSaveModel(ServicioEntity Item)
        {
            this.ServicioId = Item.ServicioId;
            this.Correlativo = Item.Correlativo;
            this.Nombre = Item.Nombre;
            this.Descripcion = Item.Descripcion;
            this.TipoServicioId = Item.TipoServicioId;
            this.FechaRegistro = Item.FechaRegistro;
            this.CodUsuario = Item.CodUsuario;
            this.EstadoRegistro = Item.EstadoRegistro;
        }
         [JsonPropertyName("ServicioId")]  public Int32 ServicioId { get; set; }
         [JsonPropertyName("Correlativo")]  public String Correlativo { get; set; }
         [JsonPropertyName("Nombre")]  public String Nombre { get; set; }
         [JsonPropertyName("Descripcion")]  public String Descripcion { get; set; }
         [JsonPropertyName("TipoServicioId")]  public Int32 TipoServicioId { get; set; }
         [JsonPropertyName("FechaRegistro")]  public DateTime FechaRegistro { get; set; }
         [JsonPropertyName("CodUsuario")]  public String CodUsuario { get; set; }
        [JsonPropertyName("EstadoRegistro")] public Boolean EstadoRegistro { get; set; }
    }
}
